regression-analysis/regression.py

```diff
@@ -0,0 +1,54 @@
+import pandas as pd
+import numpy as np
+import statsmodels.api as sm
+
+
+def import_data(filename):
+    '''
```

Comment:
Nit: For triple coded strings PEP8 recommends to always use double quoted strings: https://www.python.org/dev/peps/pep-0008/#string-quotes

I know we haven't talked about style guide much, but PEP8 is pretty standard and probably good practice to follow